docs/source/gbd2017_models/causes/neoplasms/breast_cancer_with_stage0/cancer_model.rst

```diff
@@ -0,0 +1,308 @@
++------------+---------------+---------------+----------------------------------------------------------------------+----------------------------------------------------+
+| i_BC|LCIS  | LCIS          | BC            | :math:`\frac{\text{incidence_c429}}{\text{prev_DCIS+prev_LCIS}}`     | i_BC|LCIS = i_BC /prev_(LCIS+DCIS)                 |
++------------+---------------+---------------+----------------------------------------------------------------------+----------------------------------------------------+
+
+.. list-table:: Data sources
+   :widths: 30 30 30
+   :header-rows: 1
+   
+   * - Measure
+     - Sources
+     - Notes
+   * - prev_DCIS 
+     - derived from DCIS prevalence ratio and prev_c429
+     - see below for prevalence ratio calculation
+   * - prev_LCIS
+     - derived from LCIS prevalence ratio and prev_c429
+     - see below for prevalence ratio calculation
+   * - crude-prevalence ratio 
+     - derived from external data
+     - see below for prevalence ratio calculation
+   * - prev_c429
+     - como for 2017 and forecasted for future years 2020-2040
+     - forcasted data filepath :download:`prev_c429<filepaths.csv>`
+   * - deaths_c429
+     - codcorrect for 2017 deaths
+     - future deaths 2020-2040 need to be derived from forecasted csmr_c429 and population size
+   * - csmr_c429
+     - csmr forecasted for future years 2020-2040
+     - forcasted data filepath :download:`csmr_c429<filepaths.csv>` 
+   * - incidence_c429
+     - como for 2017 and forecasted for future years 2020-2040
+     - forcasted data filepath :download:`incidence_c429<filepaths.csv>` 
+   * - Disability weights for breast cancer sequelae
+     - YLD appendix
+     - total breast cancer disability weight over all sequelae with ids s_277, s_5486, s_5489, s_279, s_280, s_5492
+   * - Population
+     - demography for 2017 
+     - mid-year population
+
+
+**Crude prevalence ratios**
+
+GBD does not give us any information on the prevalence or incidence of DCIS or LCIS. Hence we need to infer using data from another population, namely from MarketScan outpatient data from 2016 to 2017 in USA for age 15-65 only. From MarketScane, we obtain the a non-age specific ratio of DCIS prevalence and breast cancer prevalence among those tested. Applying this ratio to the prevalence of breast cancer in our population gives us an estimate of the prevalence of DCIS or LCIS in our population. 
+
+**DCIS**
+
+   - Crude prevalence ratio of DCIS = :math:`\frac{\text{proportion of DCIS among those tested}}{\text{proportion of stage 1+ breast cancer among those tested}}` =0.35
+
+   - Prevalence of DCIS 
+
+      = Crude prevalence ratio of DCIS x prevalence of breast cancer (prev_c429)
+
+**LCIS**
+
+   - Crude prevalence ratio of LCIS = :math:`\frac{\text{proportion of LCIS among those tested}}{\text{proportion of stage 1+ breast cancer among those tested}}` =0.08
```

Comment:
we'll have uncertainty for this eventually, so let's include it here to give the engineers advance warning: 0.08±.01